### PR TITLE
Trivial: Language Cleanup

### PR DESCRIPTION
**Pull request by wbarobinson**

### CODE REVIEW

**Comment by fanquake** [comment]
Thanks for the contribution. Some of these changes seem like re-writing for the sake of re-writing/updating. In any case, please [squash](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits) all your changes into a single commit with a descriptive [commit message](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#contributor-workflow).

File: doc/tor.md [hunk 1]
```diff
@@ -105,10 +105,10 @@ and, if not disabled, configured using the `-torcontrol` and `-torpassword` sett
 To show verbose debugging information, pass `-debug=tor`.
 
 Connecting to Tor's control socket API requires one of two authentication methods to be 
-configured. For cookie authentication the user running bitcoind must have write access 
-to the `CookieAuthFile` specified in Tor configuration. In some cases this is 
+configured. For cookie authentication the user running bitcoind must have write access
```
Comment (practicalswift) [inline doc/tor.md]: Seems to be some trailing whitespace here?

**Comment by practicalswift** [comment]
Concept ACK, but please squash into one commit and remove the trailing whitespace:

```
This diff appears to have added new lines with trailing whitespace.
The following changes were suspected:
diff --git a/doc/tor.md b/doc/tor.md
@@ -108,2 +108,2 @@ Connecting to Tor's control socket API requires one of two authentication method
+configured. For cookie authentication the user running bitcoind must have write access
+to the `CookieAuthFile` specified in Tor configuration. In some cases, this is
@@ -111 +111 @@ preconfigured and the creation of a hidden service is automatic. If permission p
+are seen with `-debug=tor` they can be resolved by adding both the user running Tor and
^---- failure generated from contrib/devtools/lint-whitespace.sh
```

File: doc/build-unix.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Some notes on how to build Bitcoin Core in Unix.
 
 Note
 ---------------------
-Always use absolute paths to configure and compile bitcoin and the dependencies,
+Always use absolute paths to configure and compile bitcoin-core and the dependencies,
```
Comment (MarcoFalke) [inline doc/build-unix.md]: Should say "Bitcoin Core"

File: doc/build-unix.md [hunk 3]
```diff
@@ -129,7 +129,7 @@ libqrencode (optional) can be installed with:
 
     sudo apt-get install libqrencode-dev
 
-Once these are installed, they will be found by configure and a bitcoin-qt executable will be
+Once these are installed, they will be found by configure and a Bitcoin-Qt executable will be
```
Comment (MarcoFalke) [inline doc/build-unix.md]: the executable name was correct before

File: doc/build-unix.md [hunk 4]
```diff
@@ -228,24 +228,24 @@ Hardening enables the following features:
     ET_DYN
 
 * Non-executable Stack
-    If the stack is executable then trivial stack based buffer overflow exploits are possible if
-    vulnerable buffers are found. By default, bitcoin should be built with a non-executable stack
+    If the stack is executable then trivial stack-based buffer overflow exploits are possible if
+    vulnerable buffers are found. By default, Bitcoin should be built with a non-executable stack,
```
Comment (MarcoFalke) [inline doc/build-unix.md]: Either lower case or "Bitcoin Core"

File: doc/build-unix.md (hunk 2 above)
Comment (wbarobinson) [reply doc/build-unix.md]: Good catch.

File: doc/build-unix.md (hunk 3 above)
Comment (wbarobinson) [reply doc/build-unix.md]: The documentation uses many different variants. Is it entirely system dependent? In build-unix.md, for instance, it shows up in upper and lower case.

Comment (MarcoFalke) [reply doc/build-unix.md]: It could be upper case on macOS, but I don't see a reason to change it here

**Comment by wbarobinson** [comment]
Thank you for your feedback, you have been super helpful. I have squashed the commits.

File: doc/README.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ Bitcoin Core
 
 Setup
 ---------------------
-Bitcoin Core is the original Bitcoin client and it builds the backbone of the network. It downloads and, by default, stores the entire history of Bitcoin transactions (which is currently more than 100 GBs); depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to a day or more.
+Bitcoin Core, the original Bitcoin client, builds the network's backbone. It downloads and, by default, stores the entire history of Bitcoin transactions (which is currently more than 160 GB). Depending on the speed of your computer and network connection, the synchronization process can take anywhere from a few hours to over a day.
```
Comment (Empact) [inline doc/README.md]: nit: I prefer the original first sentence. It flows a bit better.

File: doc/developer-notes.md [hunk 6]
```diff
@@ -477,34 +477,34 @@ class A
...
-    formatting mismatch, and it is easy to get wrong because of subtly similar naming
+    formatting mismatch, and it is easy to get wrong because of subtly similar naming.
 
-- Use `std::string`, avoid C string manipulation functions
+- Use `std::string`, avoid C string manipulation functions.
 
   - *Rationale*: C++ string handling is marginally safer, less scope for
-    buffer overflows and surprises with `\0` characters. Also some C string manipulations
-    tend to act differently depending on platform, or even the user locale
+    buffer overflows and fewer surprises with `\0` characters. Also some C string manipulations
```
Comment (jnewbery) [inline doc/developer-notes.md]: First sentence still sounds odd to me. How about:

```
C++ string handling is marginally safer, has less scope for buffer overflows and there are fewer surprises with `\0` characters.
```

?

File: doc/developer-notes.md [hunk 7]
```diff
@@ -788,19 +788,19 @@ RPC interface guidelines
...
+- Argument naming: use snake case `fee_delta` (and not, e.g. camel case `feeDelta`).
 
   - *Rationale*: Consistency with existing interface.
 
 - Use the JSON parser for parsing, don't manually parse integers or strings from
   arguments unless absolutely necessary.
 
   - *Rationale*: Introduces hand-rolled string manipulation code at both the caller and callee sites,
-    which is error prone, and it is easy to get things such as escaping wrong.
+    which is error prone, and it is easy to get things, such as escaping, wrong.
```
Comment (jnewbery) [inline doc/developer-notes.md]: I prefer without the extra commas here.

**Review by jnewbery — COMMENTED** [review]
seem like mostly good changes.

File: doc/README.md (hunk 5 above)
Comment (wbarobinson) [reply doc/README.md]: I disagree, but perhaps we could come up with a compromise, such as:
Bitcoin Core is the original and current reference implementation of the Bitcoin network.

File: doc/developer-notes.md (hunk 6 above)
Comment (wbarobinson) [reply doc/developer-notes.md]: Is this okay?

C++ string handling is marginally safer, with less scope for buffer overflows and fewer surprises with `\0` characters.

File: doc/developer-notes.md (hunk 7 above)
Comment (wbarobinson) [reply doc/developer-notes.md]: how about?:
which is error prone and hard to use. For instance, it is easy to incorrectly escape.

**Comment by fanquake** [comment]
Rebased in #13255.

**Pull request closed without merging**